ui/src/components/app-menu/QasAppMenu.vue

```diff
@@ -2,8 +2,8 @@
   <q-drawer v-model="model" content-class="bg-primary-contrast" :mini="miniMode" :width="230" @before-hide="beforeHide">
     <q-list class="text-primary" padding>
       <div v-for="(header, index) in items" :key="index">
-        <q-expansion-item v-if="hasChildren(header)" expand-icon="o_keyboard_arrow_down" expand-icon-class="opacity-20 text-primary" expand-separator :icon="header.icon" :label="header.label">
-          <q-item v-for="(item, itemIndex) in header.children" :key="itemIndex" v-ripple clickable :exact-active-class="activeItemClasses" :to="item.to">
+        <q-expansion-item v-if="hasChildren(header)" :active-class="activeSecondaryItemClasses" expand-icon="o_keyboard_arrow_down" expand-separator :icon="header.icon" :label="header.label" :to="header.to" :default-opened="shouldBeExpanded(header)">
```

Comment:
Por que precisa do `:to`?
Faltou lintar.

Reply:
O :to é para transformar o q-expansion em link, inclusive o active-class só funciona com ele